packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWSelectList/CWSelectList.tsx

```diff
@@ -124,6 +129,14 @@ export const CWSelectList = <
           DropdownIndicator,
           MultiValueRemove,
           Option: components?.Option || Option,
+          // eslint-disable-next-line react/no-multi-comp
+          SingleValue: (singleValueProps) => (
+            <CWSingleSelectItem
+              {...singleValueProps}
+              extraProp={showIcon}
+              handleClickCopyClipboard={handleClickCopyClipboard}
+            />
+          ),
```

Comment:
this should only be rendered when we want to show copy icon, lets add this conditionally